spec/policies/pending_profile_policy_spec.rb

```diff
@@ -29,11 +29,24 @@
       end
 
       context 'with resolved authn context result' do
-        let(:vtr) { ['C2.Pb'] }
+        let(:acr_values) do
+          Saml::Idp::Constants::AAL2_AUTHN_CONTEXT_CLASSREF +
```

Comment:
Would `Array#join` be easier on the eye? I see you have done it elsewhere.

Reply:
thanks for catching those two, i thought i had cleaned them all up. updated in https://github.com/18F/identity-idp/pull/11345/commits/234453899352538abd7195eef22c5a592fe6f526